architecture/core_concepts/routes.adoc

```diff
@@ -245,15 +245,18 @@ $ oc set env dc/router ROUTER_SYSLOG_ADDRESS=127.0.0.1 ROUTER_LOG_LEVEL=debug
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named *_tls.crt_*. If *_tls.crt_* is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if `DEFAULT_CERTIFICATE` or `DEFAULT_CERTIFICATE_PATH` are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if `DEFAULT_CERTIFICATE` is not specified.
+|`*DROP_SYN_DURING_RESTART*` |  false | Whether to enable an iptables rule to drop certain packets during a restart to provide a seamless restart.  See xref:../../install_config/router/default_haproxy_router.adoc#preventing-connection-failures-during-restarts[the install guide] for details.
 |`*EXTENDED_VALIDATION*` | `true` | If `true`, perform an additional extended validation step on all routes admitted by this router.
 |`*NAMESPACE_LABELS*` |  | A label selector to apply to namespaces to watch, empty means all.
 |`*PROJECT_LABELS*` |  | A label selector to apply to projects to watch, emtpy means all.
 |`*RELOAD_SCRIPT*` |  | The path to the reload script to use to reload the router.
 |`*ROUTER_ALLOWED_DOMAINS*` | | A comma-separated list of domains that the host name in a route can only be part of. Any subdomain in the domain can be used. Option `ROUTER_DENIED_DOMAINS` overrides any values given in this option. If set, everything outside of the allowed domains will be rejected.
 |`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | Length of time between subsequent "liveness" checks on backends. xref:time-units[(TimeUnits)]
+|`*ROUTER_CLIENT_FIN_TIMEOUT*` | 1s | Controls the fin timeout to the client connecting to the route. xref:time-units[(TimeUnits)]
 |`*ROUTER_COMPRESSION_MIME*` | "text/html text/plain text/css" | A space separated list of mime types to compress.
 |`*ROUTER_DEFAULT_CLIENT_TIMEOUT*`| 30s | Length of time within which a client has to acknowledge or send data. xref:time-units[(TimeUnits)]
 |`*ROUTER_DEFAULT_CONNECT_TIMEOUT*`| 5s | The maximum connect time. xref:time-units[(TimeUnits)]
+|`*ROUTER_DEFAULT_SERVER_FIN_TIMEOUT*` | 1s | Controls the fin timeout to the pod backing the the route.  xref:time-units[(TimeUnits)]
```

Comment:
s/fin/FIN

What does FIN stand for, or what is it? We should probably define it briefly here.

Reply:
I changed it to TCP FIN... but if you care, that's enough information.  This is a highly advanced thing to tune, and our defaults are fine.